core/tests/protractor_utils/ExplorationEditorMainTab.js

```diff
@@ -460,7 +461,8 @@ var ExplorationEditorMainTab = function() {
   // can then use to alter the state content, for example by calling
   // .appendBoldText(...).
   this.setContent = function(richTextInstructions) {
-  // Wait for browser to completely load the rich text editor.
+    browser.wait(until.invisibilityOf(postTutorialPopover), 5000);
+    // Wait for browser to time out the popover, which is 5000 ms.
     browser.wait(until.elementToBeClickable(stateEditContent), 10000,
       'stateEditContent taking too long to appear to set content')
```

Comment:
This should help with 
```bash
- Failed: stateContentEditor taking too long to appear to set content
      Wait timed out after 10022ms
```
Do you remember specifically which e2e test / suite was having this problem? @seanlip

Reply:
Yep, learnerdashboard. It was failing way too early.

Reply:
Once the whole waitForAngular thing is removed, I am expecting a higher level of consistency just like in other suite. 
This function does flake in other suite too so I made a few changes. Hopefully it is not as bad as it is currently.